testcase/sqllogictest/pkg/util/util.go

```diff
@@ -225,3 +225,40 @@ func RandomlyChangeReplicaRead(job, replicaRead string, db *sql.DB) {
 		log.Info("Do not change replica read")
 	}
 }
+
+// SetAndWaitTiFlashReplica create tiflash replicas for a table and wait it to be available or timeout
+func SetAndWaitTiFlashReplica(ctx context.Context, db *sql.DB, dbName, tableName string, replicaCount, retryCount int) error {
+	var sql string
+	if dbName != "" {
+		sql = fmt.Sprintf("alter table %s.%s set tiflash replica %d", dbName, tableName, replicaCount)
+	} else {
+		sql = fmt.Sprintf("alter table %s set tiflash replica %d", tableName, replicaCount)
+	}
+	log.Infof("begin to execute %s", sql)
+	MustExec(db, sql)
+
+	err := RunWithRetry(ctx, retryCount, time.Second, func() error {
+		instance, available := "", -1
+		if err := db.QueryRow("select INSTANCE FROM information_schema.cluster_info where type='tidb'").Scan(&instance); err != nil {
+			return err
+		}
+		query := fmt.Sprintf("select AVAILABLE from information_schema.tiflash_replica where "+
```

Comment:
if the dbName is "", the TABLE_SCHEMA condition should be removed.

Reply:
fixed.